### PR TITLE
NL: Add a tool to trim SVs that we will added to the Embeddings index

### PR DESCRIPTION
**Pull request by pradh**

### CODE REVIEW

File: tools/nl/analysis/statvar/sv_quantity_cvals.sql [hunk 1]
```diff
@@ -0,0 +1,79 @@
+WITH T AS (
+SELECT DISTINCT
+       SV.population_type AS pt,
+       SV.p1 AS p,
+       SV.v1 AS v
+FROM `datcom-store.dc_kg_latest.StatisticalVariable` AS SV
+WHERE SV.id <> SV.measured_prop AND
```
Comment (shifucun) [inline tools/nl/analysis/statvar/sv_quantity_cvals.sql:7]: Add some high level comment for each block of conditions?

File: tools/nl/analysis/statvar/run.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+cat "sv_quantity_cvals.sql" | bq query --use_legacy_sql=false \
```
Comment (shifucun) [inline tools/nl/analysis/statvar/run.sh:2]: how big is the input csv file?

File: tools/nl/analysis/statvar/data/sv_trimmed.csv
Comment (shifucun) [inline tools/nl/analysis/statvar/data/sv_trimmed.csv:1]: git lfs prevents file diff, would this be an issue for future review?

File: tools/nl/analysis/statvar/curate_nl_svs.py [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('input_sv_csv', 'data/sv_schemaful.csv',
+                    'Input SV CSV file')
+flags.DEFINE_string('output_sv_csv', 'data/sv_trimmed.csv',
+                    'Output SV file')
+flags.DEFINE_string('output_dbg', 'data/sv_trimmed_dbg_info.json',
+                    'Debug json')
+flags.DEFINE_string('quantity_csv', 'data/sv_quantity_cvals.csv',
```
[the remark this answers is not in the document]
Comment (pradh) [reply tools/nl/analysis/statvar/curate_nl_svs.py:29]: Clarified in description

File: tools/nl/analysis/statvar/curate_nl_svs.py [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+    'numberOfRooms',
+    'socioeconomicScenario',
+])
+
+_BLOCKED_CVAL_PARTS = [
+    'SchoolGrade',
+    'CarbonDioxideEquivalent20YearGlobalWarmingPotential',
+    'CarbonDioxideEquivalent100YearGlobalWarmingPotential',
+]
+
```
[the remark this answers is not in the document]
Comment (pradh) [reply tools/nl/analysis/statvar/curate_nl_svs.py]: Done.

File: tools/nl/analysis/statvar/curate_nl_svs.py [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+    if row[cp] in _qty_cprops and nv > 1:
+      # This is a quantity cprop and is not a DPV
+      # (since there are multiple values) for the
+      # same peer group (i.e., SVs minus this cval).
+      dbg_info[_ERR_QUANTITY_VALS].append(sv)
+      return False
+    if row[cp] == 'naics' and sv.startswith('dc/'):
+      dbg_info[_ERR_FILTERED_NAICS].append(sv)
+      return False
+    if nv > 50:
```
[the remark this answers is not in the document]
Comment (pradh) [reply tools/nl/analysis/statvar/curate_nl_svs.py]: Done.

**Review by pradh — COMMENTED** [review]
Thanks!

**Review by shifucun — APPROVED** [review]

**Pull request merged**